api/types/backend/backend.go

```diff
@@ -107,13 +107,14 @@ type ExecProcessConfig struct {
 // CreateImageConfig is the configuration for creating an image from a
 // container.
 type CreateImageConfig struct {
-	Repo    string
-	Tag     string
-	Pause   bool
-	Author  string
-	Comment string
-	Config  *container.Config
-	Changes []string
+	Repo               string
+	Tag                string
+	Pause              bool
+	Author             string
+	Comment            string
+	Config             *container.Config
+	Changes            []string
+	KeepReservedLabels bool
```

Comment:
Why the option?

Reply:
Oh! Good one, hmmm.. I think I originally thought it may be needed if there's uses elsewhere that need the old behavior, but I didn't do that in the end; let me remove